approved-establishment.yaml

```diff
@@ -0,0 +1,199 @@
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Activity"
+        400:
+          $ref: "./common.yaml#/components/responses/BadRequest"
+        401:
+          $ref: "./common.yaml#/components/responses/Unauthorized"
+        403:
+          $ref: "./common.yaml#/components/responses/Forbidden"
+  /approved-establishments/remark:
+    get:
+      description: Get all Approved Establishments Remarks
+      responses:
+        200:
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Remark"
+        400:
+          $ref: "./common.yaml#/components/responses/BadRequest"
+        401:
+          $ref: "./common.yaml#/components/responses/Unauthorized"
+        403:
+          $ref: "./common.yaml#/components/responses/Forbidden"
+  /approved-establishments/section:
+    get:
+      description: Get all Approved Establishments Sections
+      responses:
+        200:
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Section"
+        400:
+          $ref: "./common.yaml#/components/responses/BadRequest"
+        401:
+          $ref: "./common.yaml#/components/responses/Unauthorized"
+        403:
+          $ref: "./common.yaml#/components/responses/Forbidden"
+  /approved-establishments/section/chapter:
+    get:
+      description: Get all Approved Establishments Sections Chapters
+      responses:
+        200:
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Chapter"
+        400:
+          $ref: "./common.yaml#/components/responses/BadRequest"
+        401:
+          $ref: "./common.yaml#/components/responses/Unauthorized"
+        403:
+          $ref: "./common.yaml#/components/responses/Forbidden"
+
+
+components:
+  schemas:
+    ApprovedEstablishment:
+      type: object
+      properties:
+        establishmentId:
+          type: string
+          format: uuid
+          description: Unique identifier for the establishment
+        establishmentName:
+          type: string
+          description: Name of the establishment
+        approvalNumber:
+          type: string
+          description: Unique identifier for approval of the establishment
+        street:
+          type: string
+          description: Street address of the establishment
+        city:
+          type: string
+          description: City where the establishment is located
+        country: 
```

Comment:
can be linked to the country id once https://github.com/DEFRA/stw-api-descriptions/pull/2 is merged